source/rules/no-internal.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * @license Use of this source code is governed by an MIT-style license that
+ * can be found in the LICENSE file at https://github.com/cartant/eslint-plugin-rxjs
+ */
+
+import { Rule } from "eslint";
+import * as es from "estree";
+
+const rule: Rule.RuleModule = {
+  meta: {
+    docs: {
+      category: "RxJS",
+      description: "Disallows the importation of internals.",
+      recommended: true
+    },
+    fixable: null,
+    messages: {
+      forbidden: "RxJS imports from internal are forbidden."
+    },
+    schema: []
+  },
+  create: context => {
+    return {
+      ["ImportDeclaration Literal[value=/^rxjs\\u002finternal/]"]: (
```

Comment:
Wow. I love this!!! I don't think it needs to be a computed property though. A quoted key should be fine, AFAICT.